plasmapy/physics/quantum.py

```diff
@@ -95,12 +98,12 @@ def deBroglie_wavelength(V, particle):
 
         lambda_dBr = np.ones(V.shape) * np.inf * units.m
         indices = V.value != 0
-        lambda_dBr[indices] = h / (m*V[indices]*Lorentz_factor(V[indices]))
+        lambda_dBr[indices] = h / (m * V[indices] * Lorentz_factor(V[indices]))
 
     else:
 
-        if V == 0*units.m/units.s:
-            lambda_dBr = np.inf*units.m
+        if V == 0 * units.m / units.s:
+            lambda_dBr = np.inf * units.m
```

Comment:
wow
much readable
so pretty formulas

In all seriousness, thanks for paying attention to this!

Reply:
Hehe, no worries. It's not much effort, and I just end up instinctively doing it so it is easier to read for me 😄